tests/integration/debug_callback_view/test_views.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from django.http import HttpRequest
+
+from django_stomp.debug_callback_view.views import debug_callback_view
+from django_stomp.debug_callback_view.views import mock_ack
+from django_stomp.debug_callback_view.views import mock_nack
+from django_stomp.services.consumer import Payload
+
+fake_callback_path = "tests.unit.debug_callback_view.test_views.fake_callback"
+
+
+def fake_callback(payload: Payload):
+    return
+
+
+def test_should_return_error_when_method_not_is_post(mocker):
```

Comment:
**suggestion**: We could improve the typehint by adding `MockFixture` (`from pytest_mock import MockFixture`).